EPRN.UnitTests/Portal/Controllers/PrnsControllerTests.cs

```diff
@@ -21,85 +21,6 @@ public void Init()
             _prnsController = new PrnsController(_mockPrnService.Object);
         }
 
-        [TestMethod]
```

Comment:
Why have you deleted a load of unit tests?

Reply:
I moved these tests.

Reply:
Added some more tests.